src/components/card.tsx

```diff
@@ -0,0 +1,54 @@
+import ModalWindow from './modal-window';
+import CardRate from './card-rate';
+
+import { CameraType } from '../types/cameras';
+import { Link } from 'react-router-dom';
+import { useState } from 'react';
+
+
+type CardProps = {
+  cameraCard: CameraType;
+}
+
+function Card({cameraCard}: CardProps): JSX.Element {
+  const {name, previewImgWebp, previewImgWebp2x, previewImg, previewImg2x, rating, reviewCount, price } = cameraCard;
+  const [active, setActiv] = useState(false);
```

Comment:
```suggestion
  const [active, setActive] = useState(false);
```
не стоит на буквах экономить время